### PR TITLE
Add configuration and scripts

### PR DESCRIPTION
**Pull request by razvand**
Add Make-based setup files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,420 @@
...
+
+### Quick Setup (aka TLDR)
+
+For a quick setup, run the commands below.
+Note that you still need to install the [requirements](#requirements).
+
+For building and running everything for `x86_64`, follow the steps below:
+
+```console
+git clone https://github.com/unikraft/app-helloworld-cpp helloworld-cpp
```
Comment (RaduNichita) [inline README.md:48]: I think we need to separate the common commands (the ones with cloning the repos) from the one specific to platform / architectures.

**Review by RaduNichita — CHANGES_REQUESTED** [review]
All good with me besides that one small comment. Thanks @razvand

File: README.md (hunk 1 above)
Comment (razvand) [reply README.md:48]: I see your point. But, currently, this is how we did for all repositories. I would keep it like this, and, if we think it's a good idea, we will update it all over the place.

Comment (RaduNichita) [reply README.md:48]: Ok, fair point. Then let's merge like it is.

**Review by RaduNichita — APPROVED** [review]
All good now. 

Waiting for #13 to be merged first.

File: README.md [hunk 2]
```diff
@@ -1,49 +1,420 @@
...
+ls .config
+.config
 ```
-$ make
+
+The `.config` file will be used in the build step.
+
+#### QEMU AArch64
+
+Use the `.config.helloworld-cpp-qemu-arm64` configuration file together with `make defconfig` to create the configuration file:
```
Comment (StefanJum) [inline README.md]:
```suggestion
Use the `.config.helloworld-cpp-qemu-aarch64` configuration file together with `make defconfig` to create the configuration file:
```

File: README.md [hunk 3]
```diff
@@ -1,49 +1,420 @@
...
+
+```console
+UK_DEFCONFIG=$(pwd)/.config.helloworld-cpp-qemu-arm64 make defconfig
```
Comment (StefanJum) [inline README.md]:
```suggestion
UK_DEFCONFIG=$(pwd)/.config.helloworld-cpp-qemu-aarch64 make defconfig
```

File: README.md [hunk 4]
```diff
@@ -1,49 +1,420 @@
...
+     ```console
+     cd helloworld-cpp/
+
+     ls -F
+     ```
+
+     You will see the contents of the repository:
+
+     ```text
+     .config.helloworld-cpp-fc-x86_64  .config.helloworld-cpp-qemu-arm64  .config.helloworld-cpp-qemu-x86_64  helloworld-cpp-fc-x86_64.json  kraft.yaml  Makefile  Makefile.uk  README.md [...]
```
Comment (StefanJum) [inline README.md]:
```suggestion
     .config.helloworld-cpp-fc-x86_64  .config.helloworld-cpp-qemu-aarch64  .config.helloworld-cpp-qemu-x86_64  helloworld-cpp-fc-x86_64.json  kraft.yaml  Makefile  Makefile.uk  README.md [...]
```

**Review by StefanJum — CHANGES_REQUESTED** [review]
See the comments.

**Comment by razvand** [comment]
@StefanJum, fixed.

**Review by StefanJum — APPROVED** [review]
All good, I'll add the tag after #13 is merged and this is rebased.

**Review by StefanJum — APPROVED** [review]
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Review by RaduNichita — APPROVED** [review]
All good with rebase. Thanks, @razvand! 

Reviewed-by: Radu Nichita <radunichita99@gmail.com>

**Pull request closed without merging**